x-pack/plugins/global_search_providers/server/providers/saved_objects/provider.ts

```diff
@@ -25,7 +25,7 @@ export const createSavedObjectsResultProvider = (): GlobalSearchResultProvider =
       const responsePromise = client.find({
         page: 1,
         perPage: maxResults,
-        search: term,
+        search: term ? `${term}*` : undefined,
```

Comment:
Is there any issue with using `*${term}*` so we can match anywhere in the field? Maybe this reduces the quality of the search results.

Ideally, we'd be boosting results where the beginning of the field matches the search term, probably using multiple `should` clauses, one with the `phrase_prefix` match type. Definitely out of scope here though.

Reply:
> Is there any issue with using *${term}* so we can match anywhere in the field? Maybe this reduces the quality of the search results.

Yea, I'm not really sure. Without proper boost, searching anywhere in the field can have some unexpected behavior in term of results. I feel like until we are able to construct a more precise search dsl, prefix-only may be better.

@ryankeairns WDYT?

Reply:
I’m happy with the prefix-only change for now and collecting more feedback post-merge/release.